### ISSUE TITLE
FeatureIDE nature is lost after some time in a project using munge

### ISSUE THREAD

**Issue by joliebig** [FeatureIDE/FeatureIDE#312]
Hi,
I created a new FeatureIDE project (with munge). After some time working on the project, the FeatureIDE nature is lost the project.

1. I cannot create new configurations for the project. FeatureIDE does not recognize the project as an FeatureIDE project anymore.
2. I cannot switch the configuration. No configuration is set to current. "Set as current configuration" does nothing.
3. Going into project properties -> FeatureIDE -> Feature Project it says Project xyz is no FeatureIDE project.
4. I cannot create variants; src folder is not updated.

Creating a new project an copying the existing sources did not help this time.

I'm wondering what went wrong and how I can fix it. Thx in advance.

**Comment by RSchroeter** [comment]
The previous commit is not a bugfix but it allows us to avoid this bug in Munge. Therefore, the option "Create Signature" (Project -> FeatureIDE -> Munge -> Create Signature) have to be disabled.

**Comment by tthuem** [comment]
@RSchroeter What is the plan for this issue?

**Comment by tthuem** [comment]
Christopher, can you reproduce the bug?

**Comment by Kogoro** [comment]
I imported the munge example project and added some classes and features. I can create configs and can also switch between them. The project is always registered as FeatureIDE Munge project. Also, I have created some source files, but the update does not happen automatically. I can trigger the update manually through switching the configuration or clean the project, but the output in the project view is wrong as seen in the picture. Furthermore, the Create Signature entry is selectable.

![munge_package_explorer](https://cloud.githubusercontent.com/assets/8019776/16889751/ed523d4e-4ae8-11e6-97cd-43fcc31155c7.png)
